### PR TITLE
init html template for law firm

### PR DESCRIPTION
**Pull request by ratulkhan74**
Header, hero section and service section currently working on service section

### CODE REVIEW

File: assets/headshots-20230628T065504Z-001.zip
Comment (stratofax) [inline assets/headshots-20230628T065504Z-001.zip:1]: We don't need the zip file under source control, please remove this file.

File: assets/headshots/Christine_Baily_107DS3b.jpeg
Comment (stratofax) [inline assets/headshots/Christine_Baily_107DS3b.jpeg:1]: If you want to add all these files to the repo, these are all `photos` but only some are "headshots" so consider renaming the folder to `photos`

File: assets/~$ailylaw_com_revised_text_and_testimonials.docx
Comment (stratofax) [inline assets/~$ailylaw_com_revised_text_and_testimonials.docx:1]: This looks like a Word temp file and should be removed before the merge. Also, you may want to update the `.gitignore` file I added to the `development` branch to filter out these types of files in the future.

File: public_html/assets/imgs/christine_baily.png
Comment (stratofax) [inline public_html/assets/imgs/christine_baily.png:1]: Consider a more descriptive name, like `c_baily_standing.png` as this site will certainly have more pictures of Christine Baily

File: public_html/index.html [hunk 1]
```diff
@@ -1,151 +1,146 @@
...
+            <!-- logo -->
+            <div class="cbailylaw__logo">
+                <a href="https://www.cbailylaw.com/">C. Baily Law</a>
+            </div>
+            <!-- end of logo -->
+
+            <!-- menu -->
+            <nav class="cbailylaw__menu">
+                <ul>
+                    <li><a href="https://www.cbailylaw.com/">home</a></li>
```
Comment (stratofax) [inline public_html/index.html]: To ensure these links will work on the staging server, which will have a different domain name, please use the following format for the current directory, regardless of domain:

```
<a href="./">home</a>
```

File: public_html/index.html [hunk 2]
```diff
@@ -1,151 +1,146 @@
...
-
-<p>Thank you for visiting the website of C Baily Law LLC. Information
-offered on the website is intended for informational purposes only and
-should not be considered legal advice.</p>
+    <!-- Start header section -->
+    <header id="cbailylaw_header">
+        <div class="cbailylaw_header_inner">
+            <!-- logo -->
+            <div class="cbailylaw__logo">
+                <a href="https://www.cbailylaw.com/">C. Baily Law</a>
```
Comment (stratofax) [inline public_html/index.html]: To ensure these links will work on the staging server, which will have a different domain name, please use the following format for the current directory, regardless of domain:

```
<a href="./">home</a>
```

File: public_html/index.html [hunk 3]
```diff
@@ -1,151 +1,146 @@
...
+                            governmental actions on appeal.</p>
+                    </div>
+                    <!-- End of service card -->
+
+                </div>
+                <!-- End of service wrapper -->
+            </div>
+        </section>
+        <!-- End of service Section -->
+
```
Comment (stratofax) [inline public_html/index.html:159]: I noticed that you deleted the text from the unused sections. Another option is to put them in a temporary `div` and set the visibility to hidden, so you still have the text here in the document.

File: public_html/style.css [hunk 4]
```diff
@@ -0,0 +1,255 @@
+/* Common style css */
+:root {
+
+    /* color */
+    --color-primary: #177d89;
```
Comment (stratofax) [inline public_html/style.css]: I believe this is supposed to be `#177e89`

File: public_html/style.css [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+    --color-secondary: #f2c833;
```
Comment (stratofax) [inline public_html/style.css]: This yellow secondary color should be `#f3cb2b`

File: public_html/style.css [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+    --color-black: #000000;
+    --color-black-1e: #1e1906;
```
Comment (stratofax) [inline public_html/style.css:18]: I'm not sure how you're using this color, but I think you want `#1d2f6f`

File: public_html/style.css [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+    --color-black-gray: #3f3f3f;
```
Comment (stratofax) [inline public_html/style.css:19]: Perhaps this is `#D5DFE5`?

File: public_html/style.css [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+    --color-white: #fff;
```
Comment (stratofax) [inline public_html/style.css:20]: The white should be: `#FOEFF4`

File: public_html/style.css [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+    --color-line: #4597a1;
```
Comment (stratofax) [inline public_html/style.css:21]: I don't know about this color, whether you use it or not.

File: public_html/style.css [hunk 10]
```diff
@@ -0,0 +1,255 @@
+/* Common style css */
+:root {
+
+    /* color */
```
Comment (stratofax) [inline public_html/style.css:14]: The hex values seem slightly off here. I annotated each color entity below with my suggestions. Please check with Johan. Maybe you got these values from his Figma file?

**Review by stratofax — CHANGES_REQUESTED** [review]
Please remove the zip and Word temp files, fix the links to "home" (to "./") and double-check the hex values, as you review my comments. Thanks!

**Comment by ratulkhan74** [comment]
I have removed the zip file from the project folder also checked the color code these are same

File: public_html/index.html (hunk 1 above)
Comment (ratulkhan74) [reply public_html/index.html]: Already removed

File: assets/~$ailylaw_com_revised_text_and_testimonials.docx
Comment (ratulkhan74) [reply assets/~$ailylaw_com_revised_text_and_testimonials.docx:1]: I will remove the doc today before update my branch

File: public_html/style.css (hunk 10 above)
Comment (ratulkhan74) [reply public_html/style.css:14]: Yeah I got all color from figma file

File: public_html/style.css (hunk 9 above)
Comment (ratulkhan74) [reply public_html/style.css:21]: I have used all color you have seen in root

File: public_html/style.css (hunk 4 above)
Comment (ratulkhan74) [reply public_html/style.css]: I got the code from figma

File: public_html/assets/imgs/services/3.png
Comment (stratofax) [inline public_html/assets/imgs/services/3.png:1]: This doesn't appear to have a transparent background

File: public_html/assets/imgs/services/4.png
Comment (stratofax) [inline public_html/assets/imgs/services/4.png:1]: This doesn't appear to have a transparent background

**Review by stratofax — APPROVED** [review]
Looks great, approved!

File: public_html/assets/imgs/services/3.png
Comment (ratulkhan74) [reply public_html/assets/imgs/services/3.png:1]: i have fixed the background issue with css

**Comment by ratulkhan74** [comment]
please marge branch

**Pull request merged**